packages/grid/_modules_/grid/hooks/features/columnReorder/useGridColumnReorder.tsx

```diff
@@ -42,6 +43,7 @@ export const useGridColumnReorder = (apiRef: GridApiRef): void => {
 
   const [, setGridState, forceUpdate] = useGridState(apiRef);
   const dragCol = useGridSelector(apiRef, gridColumnReorderDragColSelector);
+  const options = useGridSelector(apiRef, optionsSelector);
```

Comment:
you can destructure 'disableColumnReorder'

Reply:
Actually, in the future, if we could refactor the other usages of `useGridSelector(apiRef, optionsSelector)` to **not** destructure, it that it would be great.

It will make it easier to find how the props are used in the codebase (allowing to grep `option.x` instead of `x`.